presto-orc/src/main/java/io/prestosql/orc/OrcWriterOptions.java

```diff
@@ -117,39 +132,62 @@ public DataSize getMaxCompressionBufferSize()
         return maxCompressionBufferSize;
     }
 
+    public double getBloomFilterFpp()
+    {
+        return bloomFilterFpp;
+    }
+
+    public Supplier<BloomFilterBuilder> getBloomFilterBuilder(String columnName)
```

Comment:
The `Options` class isn't really intended to be factory, but a simple data-only struct.  I move this code to the caller, and add a `boolean isBloomFilterColumn(String columnName)` here instead.